frontend/src/components/pages/NewAccountPage.tsx

```diff
@@ -50,6 +98,16 @@ const NewAccountPage = (): React.ReactElement => {
         employee,
       },
     });
+
+    if (!createEmployeeError) {
+      await deleteUserInvite({
+        variables: {
+          email: userInvite?.email,
+        },
+      });
+    }
+
+    history.push("/account-created");
```

Comment:
Would this make more sense?
```suggestion
    if (!createEmployeeError) {
      await deleteUserInvite({
        variables: {
          email: userInvite?.email,
        },
      });
    
      history.push("/account-created");
    }
```